telegram-bot-itmo/plugins/practiceplugin/src/main/kotlin/ru/vladrus13/itmobot/plugin/practice/PracticePlugin.kt

```diff
@@ -13,15 +16,26 @@ class PracticePlugin : Plugin() {
     override val isAvailableUser = true
     override val isAvailableChat = false
 
-    override fun getDataBases(): List<Pair<KClass<*>, DataBaseEntity<*>>> = listOf()
+    override fun getDataBases(): List<Pair<KClass<*>, DataBaseEntity<*>>> =
+        listOf(Pair(SheetJob::class, SheetJobParser()))
 
-    override fun init() {}
+    override suspend fun init() {
+        try {
+            while (true) {
```

Comment:
https://kotlinlang.org/docs/cancellation-and-timeouts.html#making-computation-code-cancellable